graffiti/src/com/nocircleno/graffiti/interaction/TouchInteractionEventManager.as

```diff
@@ -125,7 +125,7 @@ package com.nocircleno.graffiti.interaction
          {
             _target.removeEventListener(Event.ENTER_FRAME, drawToFrame);
             
-            if(_interactionCompleteCallback)
+            if(_interactionCompleteCallback())
```

Comment:
This seems weird.  The point of this is to check to see if a callback function has been assigned.  Calling ` if(_interactionCompleteCallback)` on a function that has been assigned must not return a truthy value.  I did some testing and change this line to `if(_interactionCompleteCallback != null) {` and all should be right.